website_livechat_firstname/static/description/index.html

```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
```

Comment:
No need for this file, the `README.rst` handles this except in some incredibly specific circumstances you probably won't see for a while